### PR TITLE
add scrollbar for LazyVerticalGrid

### PR DESCRIPTION
**Pull request by walt-zhong**
add scrollbar for lazyVerticalGrid.any questions,contact me please!!

### CODE REVIEW

File: lib/src/main/java/my/nanihadesuka/compose/LazyGridVerticalScrollbar.kt [hunk 1]
```diff
@@ -0,0 +1,383 @@
...
+import kotlinx.coroutines.delay
+import kotlinx.coroutines.launch
+import kotlin.math.floor
+
+/**
+ * @Author: walt-zhong
+ * @Date: 2023/11/1
+ * @Email: 2511255880@qq.com
+ * @Description:scroller for LazyVerticalGrid
+ */
```
Comment (nanihadesuka) [inline lib/src/main/java/my/nanihadesuka/compose/LazyGridVerticalScrollbar.kt]: delete this please

**Review by nanihadesuka — CHANGES_REQUESTED** [review]
@walt-zhong PR looks good.

For approval the PR needs UI unit tests for the lazy grid scrollbar. You can create a new unit test file with all the cases of the lazy list and add the special ones for the grid list you feel are appropriate.

File: lib/src/main/java/my/nanihadesuka/compose/LazyGridVerticalScrollbar.kt (hunk 1 above)
Comment (walt-zhong) [reply lib/src/main/java/my/nanihadesuka/compose/LazyGridVerticalScrollbar.kt]: ok

**Comment by walt-zhong** [comment]
I will create a new one pr

**Pull request closed without merging**